wcomponents-core/src/main/java/com/github/bordertech/wcomponents/WMessageBox.java

```diff
@@ -164,8 +167,12 @@ public Type getType() {
 	 * @param args optional arguments for the message format string.
 	 */
 	public void setTitleText(final String title, final Serializable... args) {
-		MessageModel model = getOrCreateComponentModel();
-		model.title = I18nUtilities.asMessage(title, args);
+		String currTitle = getTitleText();
```

Comment:
Same as before, this should test the response from`asMessage` against the current value.